client/src/components/SiteForm.js

```diff
@@ -0,0 +1,162 @@
+import React, { useState } from "react";
+import translateServerErrors from "../services/translateServerErrors.js";
+import { Redirect } from "react-router-dom";
+import ErrorList from "./layout/ErrorList";
+import Dropzone from "react-dropzone";
+
+const SiteForm = (props) => {
+    const { user } = props;
+    const [siteRecord, setSiteRecord] = useState({
+        name: "",
+        creatorId: user.id,
```

Comment:
this should be handled on the back end as express and passport are keeping track of the currently signed in user through` req.user`. the front end form should only store information provided by the user